src/Util/Animation.cpp

```diff
@@ -8,8 +8,7 @@ Animation::Animation(const std::vector<std::string> &paths, bool play,
     : m_State(play ? State::PLAY : State::PAUSE),
       m_Interval(interval),
       m_Looping(looping),
-      m_Cooldown(cooldown),
-      m_Index(0) {
+      m_Cooldown(cooldown) {
```

Comment:
doesn't this constructor need to initialize `m_TimeBetweenFrameUpdate`?
IIRC only the default constructor does default member initialization

Reply:
But if I use constructor initialize list to initialize ```m_TimeBetweenFrameUpdate```,
SonarCloud checks won't pass.

Reply:
you can ignore sonarcloud warnings

Reply:
What? I thought that sonarcloud checks don't pass mean can't merge.
Should I change back?

Reply:
we allow merging without all checks passing, you can decide if you want to change it back or not

Reply:
ok, that me keep it now.